.azure-pipelines/statusChain.py

```diff
@@ -0,0 +1,58 @@
+import time
+
+TestPlanStatus = {
```

Comment:
please use enum instead of a dict, that will be a more standard way

Reply:
If we use enum, we can not get the value of status using TestPlanStatus.string or TestPlanStatus[string]